README.md

```diff
@@ -6,6 +6,12 @@ List of GitHub profiles that have awesome customisation, that you can use for in
 
 Feel free to submit a pull request with your suggestions of awesome github profiles.
 
+[![made with love](https://forthebadge.com/images/badges/built-with-love.svg)](https://github.com/EddieHubCommunity)
+[![made with markdown](https://forthebadge.com/images/badges/made-with-markdown.svg)](https://github.com/EddieHubCommunity)
+[![open source](https://forthebadge.com/images/badges/open-source.svg)](https://github.com/EddieHubCommunity)
+[![check it out](https://forthebadge.com/images/badges/check-it-out.svg)](https://github.com/EddieHubCommunity)
+[![forthebadge](https://forthebadge.com/images/badges/built-by-developers.svg)](https://github.com/EddieHubCommunity)
```

Comment:
IMO, this badge should appear in the center. It would look cooler.
Let me know if you need any help with that. 🙂

Reply:
@Panquesito7 , do you mean we should align them to the center of the page? if so, I can add the html code and set the align to center.

Reply:
> @Panquesito7 , do you mean we should align them to the center of the page? if so, I can add the html code and set the align to center.

It doesn't seem to work:

<img src="https://forthebadge.com/images/badges/made-with-markdown.svg" style="align-items:center;">
I put `align-items:center`